File: alphaJava/f11_Function/assign5.java
public class assign5 {
    public static int sum(int n){
        int sum =0;
        int ld;
        while (n>0){
            ld = n%10;
            sum += ld;
            n/=10;
        }
        return sum;
    }   
    public static void main(String args[]){
        System.out.println(sum(1238));
    } 
}

File: Cpp/August/week3/loops_2_assignment/assign5.cpp
#include <iostream>
using namespace std;
int main(){
    int n = 4556;
    int lastDigit;
    int sum = 0;
    while(n>0){
        lastDigit = n % 10;
        n/=10;
        if(lastDigit % 2 ==0){
            sum += lastDigit;
        }
    }
    cout << sum;
}